### PR TITLE
- Add pipeline strategy for Nice clusters env and Be env.

### PR DESCRIPTION
**Pull request by avlaropereira**
- Intrduce a StrategyManager in order to trigger different pipelines logic fot both NiceIncontact and Brand Embassy Environments.

### CODE REVIEW

File: pipelines/beAgentCreationPipeline.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+var Pipeline = require('./pipeline');
+class BrandEmbassyUserCreationPipeline extends Pipeline {
+	constructor(params) {
+		super(params);
+	}
+
+	initialize() {
+		var BrandEmbassyAttachUser = require('../requesters/creator/brandEmbassy/userAttach');
```
Comment (IvanMoralesCamacho) [inline pipelines/beAgentCreationPipeline.js]: this should be in the top of the class

File: requesters/creator/brandEmbassy/userAttach.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const HttpRequester = require('../../httpRequester');
+const fs = require('fs');
```
Comment (IvanMoralesCamacho) [inline requesters/creator/brandEmbassy/userAttach.js]: move those variables inside the class

File: pipelines/beAgentCreationPipeline.js (hunk 1 above)
Comment (avlaropereira) [reply pipelines/beAgentCreationPipeline.js]: It has been moved .... thanks for notice it.

File: requesters/creator/brandEmbassy/userAttach.js (hunk 2 above)
Comment (avlaropereira) [reply requesters/creator/brandEmbassy/userAttach.js]: all those variable has been moved inside class ... thanks for the observation.

**Pull request merged**